app/models/site_config.rb

```diff
@@ -53,6 +53,8 @@ class SiteConfig < RailsSettings::Base
   field :staff_user_id, type: :integer, default: 1
   field :experience_low, type: :string, default: "Total Newbies"
   field :experience_high, type: :string, default: "Experienced Users"
+  field :tag_feed_minimum_score, type: :integer, default: 1
+  field :minimum_hotness_score, type: :integer, default: 7
```

Comment:
I think the best, possibly long term but definitely short term, solution is to make this hotness score configurable. I am _pretty sure_ that the issue in smaller Forems is that `7` is too high, and it needs to be adjusted to something lower for now.

Therefore, smaller Forems will need to set this in console to something lower:
```
SiteConfig.minimum_hotness_score = 5 # or 3?
```